evm/utils.js

```diff
@@ -777,6 +778,57 @@ function getConfigByChainId(chainId, config) {
     throw new Error(`Chain with chainId ${chainId} not found in the config`);
 }
 
+const addEnvironmentOptions = (program, ignoreChainNames) => {
+    program.addOption(
+        new Option('-e, --env <env>', 'environment')
+            .choices(['local', 'devnet', 'stagenet', 'testnet', 'mainnet'])
+            .default('testnet')
+            .makeOptionMandatory(true)
+            .env('ENV'),
+    );
+
+    if (!ignoreChainNames) {
+        program.addOption(
+            new Option('-n, --chainNames <chainNames>', 'chains to run the script over').makeOptionMandatory(true).env('CHAINS'),
+        );
+    }
+
+    return program;
+};
+
+const addDeploymentOptions = (program, artifactPath, contractName, salt, skipChains, skipExisting, upgrade) => {
+    addEnvironmentOptions(program);
+
+    program.addOption(new Option('-p, --privateKey <privateKey>', 'private key').makeOptionMandatory(true).env('PRIVATE_KEY'));
+    program.addOption(new Option('-v, --verify', 'verify the deployed contract on the explorer').env('VERIFY'));
+    program.addOption(new Option('-y, --yes', 'skip deployment prompt confirmation').env('YES'));
+
+    if (artifactPath) program.addOption(new Option('-a, --artifactPath <artifactPath>', 'artifact path'));
+    if (contractName) program.addOption(new Option('-c, --contractName <contractName>', 'contract name').makeOptionMandatory(true));
+    if (salt) program.addOption(new Option('-s, --salt <salt>', 'salt to use for create2 deployment').env('SALT'));
+    if (skipChains) program.addOption(new Option('--skipChains <skipChains>', 'chains to skip over'));
+
+    if (skipExisting) {
+        program.addOption(new Option('-x, --skipExisting', 'skip existing if contract was already deployed on chain').env('SKIP_EXISTING'));
+    }
+
+    if (upgrade) program.addOption(new Option('-u, --upgrade', 'upgrade a deployed contract').env('UPGRADE'));
+
+    return program;
+};
+
+const addCallContractOptions = (program, skipChains) => {
+    addEnvironmentOptions(program);
+
+    program.addOption(new Option('-a, --address <address>', 'override address'));
+    program.addOption(new Option('-p, --privateKey <privateKey>', 'private key').makeOptionMandatory(true).env('PRIVATE_KEY'));
+    program.addOption(new Option('-y, --yes', 'skip deployment prompt confirmation').env('YES'));
+
+    if (skipChains) program.addOption(new Option('--skipChains <skipChains>', 'chains to skip over'));
```

Comment:
skip chains should be added in the base env options. it was missing in some scripts before